core/src/integration-test/java/io/aiven/kafka/tieredstorage/RemoteStorageManagerTest.java

```diff
@@ -86,6 +91,11 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
```

Comment:
Should we have a separate unit test for RemoteStorageManager and move pure unit tests there? And rename this one to e.g. RemoteStorageManagerIT? Looks weird to have mocks in integration-test directory.

Reply:
I moved the new tests to the normal `test` dir.